src/PostgreSQL.xml

```diff
@@ -18,14 +18,14 @@
       <p>Permission to use, copy, modify, and distribute this software and its documentation for any purpose,
          without fee, and without a written agreement is hereby granted, provided that the above copyright
          notice and this paragraph and the following two paragraphs appear in all copies.</p>
-      <p>IN NO EVENT SHALL THE UNIVERSITY OF CALIFORNIA BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT, SPECIAL,
+      <p>IN NO EVENT SHALL <alt match=".+" name="copyrightHolderLiability">THE UNIVERSITY OF CALIFORNIA</alt> BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT, SPECIAL,
          INCIDENTAL, OR CONSEQUENTIAL DAMAGES, INCLUDING LOST PROFITS, ARISING OUT OF THE USE OF THIS SOFTWARE
-         AND ITS DOCUMENTATION, EVEN IF THE UNIVERSITY OF CALIFORNIA HAS BEEN ADVISED OF THE POSSIBILITY OF
+         AND ITS DOCUMENTATION, EVEN IF <alt match=".+" name="copyrightHolderLiability">THE UNIVERSITY OF CALIFORNIA</alt> HAS BEEN ADVISED OF THE POSSIBILITY OF
```

Comment:
You're reusing `copyrightHolderLiability` here.  I'm not aware of a clear policy on `<alt>` name uniqueness, but we have a number of examples that go out of their way to use unique names (e.g. [here][1]).  We also have cases like the MPL-1.1, where [we do not use unique names][2].  Until we get a clear policy on this, I've been erring on the side of unique names.

[1]: https://github.com/spdx/license-list-XML/blob/9f4432fbb660510859417b3d78a795beeeb8279b/src/HPND.xml#L41-L43
[2]: https://github.com/spdx/license-list-XML/blob/9f4432fbb660510859417b3d78a795beeeb8279b/src/MPL-1.1.xml#L20-L28